pyDeltaRCM/init_tools.py

```diff
@@ -109,6 +109,11 @@ def import_files(self, kwargs_dict={}):
             else:
                 default_dict[k]['type'] = [eval(_v) for _v in v['type']]
 
+        # merge optional subclass params into defaults (subclass supercedes)
+        if hasattr(self, '_subclass_parameters'):
+            default_dict.update(self._subclass_parameters)
+            delattr(self, '_subclass_parameters')  # dont need attr anymore
```

Comment:
I would advise against deleting it, I think. It's a harmless variable, and deleting it *could* be more confusing that just leaving it around. Idk, maybe someone wants the list to check for the fact that their custom vars have changed or something